### PR TITLE
adding list of min permissions

### PR DESCRIPTION
**Pull request by prat98**

### CODE REVIEW

File: stories/topics/ref-aap-gcp-install-prerequisites.adoc [hunk 1]
```diff
@@ -24,4 +24,90 @@ You must have a working knowledge of the following aspects of {GCP}:
 ** GCP Ops agent
 * SSH
 
+In addition to this, your GCP account must have the permissions listed below to successfully create an {PlatformNameShort} deployment on _{GCP}_.
```
Comment (ianf77) [inline stories/topics/ref-aap-gcp-install-prerequisites.adoc:27]: In addition, your

File: stories/topics/ref-aap-gcp-install-prerequisites.adoc [hunk 2]
```diff
@@ -24,4 +24,90 @@ You must have a working knowledge of the following aspects of {GCP}:
...
+
```
Comment (ianf77) [inline stories/topics/ref-aap-gcp-install-prerequisites.adoc:28]: Take out this line.

**Review by ianf77 — COMMENTED** [review]
Two comments, but a bit of a change required.

File: stories/topics/ref-aap-gcp-install-prerequisites.adoc (hunk 1 above)
Comment (prat98) [reply stories/topics/ref-aap-gcp-install-prerequisites.adoc]: Yeah that makes sense! Made the updates you requested

File: stories/topics/ref-aap-gcp-install-permissions.adoc [hunk 3]
```diff
@@ -0,0 +1,88 @@
+[id="ref-aap-gcp-install-permissions"]
```
Comment (ianf77) [inline stories/topics/ref-aap-gcp-install-permissions.adoc:1]: Don't forget you'll have to add this file to the assembly.

assembly-aap-gcp-install.adoc

Add it after this line

include::topics/ref-aap-gcp-install-prerequisites.adoc[leveloffset=+1]
in the following form
include::topics/ref-aap-gcp-install-permissions.adoc[leveloffset=+1]

(make it +2 if you want it to be a subsection of the Prerequisites.)

**Review by ianf77 — COMMENTED** [review]
One more vital comment

**Review by ianf77 — APPROVED** [review]
LGTM

Shall I merge?

**Comment by prat98** [comment]
Yes you can merge

**Pull request merged**